torchvision/datasets/eurosat.py

```diff
@@ -53,7 +56,7 @@ def download(self) -> None:
 
         os.makedirs(self._base_folder, exist_ok=True)
         download_and_extract_archive(
-            "https://madm.dfki.de/files/sentinel/EuroSAT.zip",
+            "https://huggingface.co/datasets/torchgeo/eurosat/resolve/c877bcd43f099cd0196738f714544e355477f3fd/EuroSAT.zip",
```

Comment:
Could use `hf.co` or `main` to make this URL shorter if you want. I don't plan on modifying this file, but the commit hash ensures that even if I do, the checksum still works.